contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_checker.py

```diff
@@ -121,21 +125,3 @@ def test_syntax_error_nit(self):
                      """     |invalid python\n"""
                      """     |""",
                      str(nits[0]))
-
-  def test_multiline_nit_printed_only_once(self):
```

Comment:
I couldn't think of an alternative test - it seems like the original was more related to print logic than checker logic? 

Let me know if you have ideas.